### PR TITLE
Quarkus Live Update with tilt

### PR DESCRIPTION
**Pull request by bandrade**
I would like to add this example that is using [Quarkus](https://quarkus.io/). 

Please have review

### CODE REVIEW

File: 201-quarkus-live-update/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- Have [tilt](https://docs.tilt.dev/install.html) properly installed and a kubernetes environment. Ensure that you have `KUBECONFIG` environment var or is in the proper kube context.
+- Have Docker or Buildah properly installed.
+
+## Running the demo
+
+Clone the demo sources locally:
+
+```
+git clone https://github.com/tilt-dev/tilt-example-java.git
+cd 201-quarkus-live-update/
```
Comment (nicks) [inline 201-quarkus-live-update/README.md]: i think this is supposed to be

```
cd tilt-example-java/201-quarkus-live-update
```

**Review by nicks — COMMENTED** [review]
Nice! I played around with this a bit. It's very fast!

A couple clarification questions (in case anyone asks us about it):

1) The trick to use a Persistent Volume for maven caching is clever! But I don't think that's how we would usually advise people to do it. When you `tilt down` your build, you lose the cache. Is there a mvn command to download the deps as part of the docker build?

2) Would it make sense to have a quarkus extension?
https://github.com/tilt-dev/tilt-extensions
That would make it a bit easier to have people re-use the logic. We should definitely have a buildah extension. (I think it still makes sense to have a working example like this, even if we also have an extension)

**Comment by bandrade** [comment]
> Nice! I played around with this a bit. It's very fast!
> 
> A couple clarification questions (in case anyone asks us about it):
> 
> 1. The trick to use a Persistent Volume for maven caching is clever! But I don't think that's how we would usually advise people to do it. When you `tilt down` your build, you lose the cache. Is there a mvn command to download the deps as part of the docker build?

`mvn dependency:go-offline` can be used to that matter, I think that would be nice to have everything in the image. But as far as I tested if you run `tilt down` and `tilt up` again, the container build would be triggered again, so I think we would have the same steps but in different orders. Please correct me if I'm wrong.

> 2. Would it make sense to have a quarkus extension?
>    https://github.com/tilt-dev/tilt-extensions
>    That would make it a bit easier to have people re-use the logic. We should definitely have a buildah extension. (I think it still makes sense to have a working example like this, even if we also have an extension)

I think that would be nice since some frameworks have different ways to do the same thing, I guess that would help users to identify how to do it with an extension

File: 201-quarkus-live-update/README.md (hunk 1 above)
Comment (bandrade) [reply 201-quarkus-live-update/README.md]: Added a new commit with the correction. Thanks.

**Comment by nicks** [comment]
thanks!

**Pull request merged**